prunning_implementation/prunning.py

```diff
@@ -0,0 +1,99 @@
+import babeltrace
+from babeltrace import CTFWriter
+import sys
+import tempfile
+from collections import Counter
+from collections import deque
+
+
+#############################################################################
+#    Function to Prunn the data and put in a deque                          #
+#    @param trace_path: The path to the stream file                #
+#           capacity: The capacity of the deque                             #
+############################################################################
+
+def getPrunnedData(trace_path, capacity):
+    
+    myDeque = deque(maxlen = capacity)
+    trace_collection = babeltrace.TraceCollection()
+    
+    # Raise an exception
```

Comment:
this is not a particularly useful comment since it does not always raise an exception here (thankfully). the code is clear enough here, just eliminate the comment.